.github/workflows/trigger-preview.yaml

```diff
@@ -8,24 +8,7 @@ on:
       - completed
 
 jobs:
-  automate-metrics:
-    runs-on: macos-latest
-    steps:
-      - uses: actions/checkout@v3
-      - name: Automate Metrics
-        env:
-          PRIVATE_KEY: ${{ secrets.PRIVATE_KEY }}
-          PRIVATE_KEY_ID: ${{ secrets.PRIVATE_KEY_ID }}
-        run: |
-          python -m venv analytics-api
-          source analytics-api/bin/activate
-          pip install google-analytics-data cartopy matplotlib
-
-          python .github/workflows/get-metrics.py
-          python .github/workflows/write-metrics-md.py
-
   find-pull-request:
-    needs: automate-metrics
```

Comment:
I am hesitant about this removal, i.e. I think keeping the "automate-metrics" job in the "trigger-preview.yaml" workflow would still be valuable to see how the Metrics page would render in the preview of any PR. This is because right after any PR is merged into main, there will be a publish-site workflow run, which will update the metrics on the website.

Reply:
I think I was confused about each action a bit. I agree and will re-introduce this - but my attempts at introducing an artifact have [broken the actions on my branch](https://github.com/jukent/projectpythia.github.io/actions/runs/8377667084).

I think it might be better for me to focus on getting one action working (the publish-site), and then the knowledge I gain should transfer to setting up the trigger-preview to use automate-metrics again.

Reply:
Okay I added this back for now, but I know it won't work yet and I'm not sure if it can work.

I think this because the trigger-preview is grabbing a book-build artifact. 
`artifact_name: book-zip-${{ needs.find-pull-request.outputs.number }}` that is zipped from the trigger-site-build action (which is triggered by a PR and thus does not have access to the secrets necessary to run the metrics collection). And in turn the trigger-preview is triggered by the trigger-site-build. 

There seems to be a possible work around with trigger-preview being allowed to access secrets, so we could rebuild the book in that step, but that seems computationally expensive. I have to think about this more.

Reply:
Removed again per our discussion

Reply:
I'm wondering if there is a benefit to moving the automate-metrics step to Cookbook-Actions even though we don't intend for it to be re-usable across repositories. It seems that if it existed as a re-usable workflow there are different permissions. For example, it could access secrets that live in the Cookbook-Actions repo and be triggered by a PR in the portal, zip up it's artifact and pass that back to the trigger-preview workflow. It would be a bit of an overall, and would be the next step after the artifact passing is functioning and well understood.

Reply:
Maybe we can revisit this in the future?